common/models/create_task_request.go

```diff
@@ -7,9 +7,10 @@ import (
 )
 
 type CreateTaskRequest struct {
-	TaskName      string `json:"task_name"`
-	EnvironmentID string `json:"environment_id"`
-	DeployID      string `json:"deploy_id"`
+	ContainerOverrides []ContainerOverride `json:"container_overrides"`
```

Comment:
We need to add the `container_overrides` field into the swagger definition below